src/internal/remote/ApiClient.ts

```diff
@@ -0,0 +1,87 @@
+import { BKTException } from '../../BKTExceptions'
+import { GetEvaluationsRequest } from '../model/request/GetEvaluationsRequest'
+import { GetEvaluationsResponse } from '../model/response/GetEvaluationsResponse'
+import { SourceID } from '../model/SourceID'
+import { User } from '../model/User'
+import { FetchLike, FetchRequestLike } from './fetch'
+import { postInternal } from './post'
+import { GetEvaluationsFailure, GetEvaluationsResult, GetEvaluationsSuccess } from './GetEvaluationsResult'
+
+export interface ApiClient {
+  getEvaluations(user: User, userEvaluationsId: string, timeoutMillis?: number): Promise<GetEvaluationsResult>
+}
+
+const DEFAULT_REQUEST_TIMEOUT_MILLIS = 30_000
+
+export class ApiClientImpl implements ApiClient {
+
+  constructor(
+    private readonly endpoint: string,
+    private readonly apiKey: string,
+    private readonly featureTag: string,
+    private readonly fetch: FetchLike,
+    private readonly defaultRequestTimeoutMillis: number = DEFAULT_REQUEST_TIMEOUT_MILLIS,
+  ) {
+
+  }
+
+  async getEvaluations(
+    user: User,
+    userEvaluationsId: string,
+    timeoutMillis: number = this.defaultRequestTimeoutMillis
+  ): Promise<GetEvaluationsResult> {
+    const body: GetEvaluationsRequest = {
+      tag: this.featureTag,
+      user,
+      user_evaluations_id: userEvaluationsId,
+      source_id: SourceID.ANDROID
```

Comment:
Please change it to `SourceID.WEB`

Reply:
[`7502934` (#13)](https://github.com/bucketeer-io/javascript-client-sdk/pull/13/commits/7502934112f931f98b9a23538c91490e954b1f1e)